src/topology/sheaves/sheafify.lean

```diff
@@ -124,6 +120,131 @@ The isomorphism betweeen a stalk of the sheafification and the original stalk.
 def sheafify_stalk_iso (x : X) : F.sheafify.1.stalk x ≅ F.stalk x :=
 (equiv.of_bijective _ ⟨stalk_to_fiber_injective _ _, stalk_to_fiber_surjective _ _⟩).to_iso
 
--- PROJECT functoriality, and that sheafification is the left adjoint of the forgetful functor.
+open category_theory
+
+instance stalk_to_fiber_is_iso (x : X) : is_iso (F.stalk_to_fiber x) :=
+(is_iso_iff_bijective _).mpr
+  ⟨stalk_to_fiber_injective _ _, stalk_to_fiber_surjective _ _⟩
+
+@[simp, reassoc] lemma stalk_functor_sheafify_comp_stalk_to_fiber (x : X) :
+  (stalk_functor (Type v) x).map F.to_sheafify ≫ F.stalk_to_fiber x = 𝟙 _ :=
+begin
+  ext U s,
+  induction U using opposite.rec,
+  cases U,
+  delta stalk_functor to_sheafify stalk_to_fiber Top.stalk_to_fiber presheaf.germ sheafify,
+  simp,
+  congr
+end
+
+@[simp, reassoc] lemma stalk_to_fiber_comp_stalk_functor_sheafify (x : X) :
+  F.stalk_to_fiber x ≫ (stalk_functor (Type v) x).map F.to_sheafify = 𝟙 _ :=
+is_iso.inv_eq_of_inv_hom_id (stalk_functor_sheafify_comp_stalk_to_fiber F x) ▸
+      is_iso.hom_inv_id _
+
+/-- A morphsim of presheaves `F ⟶ G`, can be lifted to a morphism of sheaves `F♯ ⟶ G♯`. -/
+def sheafify_map {F G : presheaf (Type v) X} (α : F ⟶ G) :
+  F.sheafify ⟶ G.sheafify :=
+{ app := λ U s, ⟨λ x, (stalk_functor _ x.val).map α (s.val x),
+  begin
+    intro x,
+    rcases s.2 x with ⟨V, hV, i, a, ha⟩,
+    refine ⟨V, hV, i, (α.app _) a, _⟩,
+    intro y, have : s.1 _ = _ := ha y,
+    dsimp only,
+    rw this,
+    erw functor.comp_map,
+    rw whiskering_left_obj_map,
+    erw limits.types.colimit.ι_map_apply,
+    congr
+  end⟩,
+  naturality' := λ A B f,
+  begin
+    ext y, cases y,
+    delta presheaf.sheafify subsheaf_to_Types subpresheaf_to_Types presheaf.stalk_functor,
+    simp only [types_comp_apply, subtype.coe_mk],
+    congr
+  end }
+
+variable (X)
+
+/-- The sheafification functor from presheaves to sheaves -/
```

Comment:
Maybe clarify that this is only (pre)sheaves of types (and not groups/rings/etc).
```suggestion
/-- The sheafification functor from presheaves of types to sheaves of types. -/
```